docs/tutorial/part-four/index.md

```diff
@@ -563,11 +552,11 @@ _about_ files but what if you want to query data _inside_ files?
 To make this possible, Gatsby supports transformer plugins which take raw
 content from source plugins and _transform_ it into something more usable.
 
-For example, markdown files. Markdown is nice to write in but when you build a
-page with it, you need the markdown to be HTML.
+For example, Markdown files. Markdown is nice to write in but when you build a
```

Comment:
"markdown" should only be capitalized when used as a noun? Maybe? So when an adjective "markdown files" it's not capitalized?

Reply:
Ah never mind! Proper adjectives are capitalized http://englishplus.com/grammar/00000048.htm

TIL